src/D2L.CodeStyle.Analyzers/Immutability/KnownImmutableTypes.cs

```diff
@@ -111,8 +111,7 @@ internal KnownImmutableTypes( IAssemblySymbol a )
 
 			foreach( var attribute in typesAttributes ) {
 				var typeofArgument = attribute.ConstructorArguments[ 0 ];
-				var value = typeofArgument.Value as INamedTypeSymbol;
-				if( value == null ) {
+				if( !( typeofArgument.Value is INamedTypeSymbol value ) ) {

```

Comment:
Some minor optimization suggested by Visual Studio.

Reply:
That is a more natural way to read the same idea.  Cool.